generators/server/templates/src/main/resources/config/_application-prod.yml

```diff
@@ -48,6 +48,9 @@ spring:
         username: <%= baseName %>
         <%_ } _%>
         password:
+        <%_ if (prodDatabaseType == 'postgresql') { _%>
```

Comment:
This was removed on purpose. Please use the DatabaseBuilder. See _DatabaseConfiguration.java from the main generator.